test/onigumo_cli_test.exs

```diff
@@ -23,8 +28,10 @@ defmodule OnigumoCLITest do
       assert usage_message_printed?(fn -> Onigumo.CLI.main(["Downloader", "Parser"]) end)
     end
 
-    test("run CLI with invalid switch") do
-      assert usage_message_printed?(fn -> Onigumo.CLI.main(["--invalid"]) end)
+    for switch <- @invalid_switches do
+      test("run CLI with invalid switch #{switch}") do
```

Comment:
Let’s use `inspect` for consistency. It makes it clear that’s an inserted variable.

```suggestion
      test("run CLI with invalid switch #{inspect(switch)}") do
```